documentation/ranking.html

```diff
@@ -5,14 +5,16 @@
 
 <p>
 Vespa uses <a href="reference/ranking-expressions.html">ranking expression</a>
-to rank documents matching a query, computing a rank score per document.
-The results of the ranking computation can be returned with the documents,
-and used to order and further select the documents that should be returned.
-Simple example: Rank the freshest document highest, using the value in an attribute named <em>timestamp</em>:
+to rank documents matching a query, computing a rank score per document using <a href="#rank-features">rank features</a>.
```

Comment:
using ranking expressions, not rank features, but an expression will likely use rank-features.